shared/src/business/useCases/courtIssuedDocument/serveCourtIssuedDocumentInteractor.js

```diff
@@ -211,6 +224,15 @@ exports.serveCourtIssuedDocumentInteractor = async (
     caseToUpdate: caseEntity,
   });
 
+  await applicationContext
+    .getPersistenceGateway()
+    .updateDocketEntryPendingServiceStatus({
+      applicationContext,
+      docketEntryId: courtIssuedDocument.docketEntryId,
+      docketNumber: caseToUpdate.docketNumber,
+      status: false,
+    });
```

Comment:
a little curious why we unset this here?

and also a little curious if we should be unsetting it in case we catch an error while initiating service so it doesn't have to be manually unset in order to re-attempt.

Reply:
@matthopson can you please look into this?

Reply:
This is to prevent all the previous steps from running in the event of an attempt to double serve this document. I did make a not in the original PR about the possibility of something being stuck in some state. I proposed wrapping it in a try/catch to reset its status and got no discussion on it.

Reply:
Do you think that's a reasonable approach? If we throw an error, unset the flag, and rethrow?

Reply:
> Do you think that's a reasonable approach? If we throw an error, unset the flag, and rethrow?

I think so.

> This is to prevent all the previous steps from running in the event of an attempt to double serve this document.

I guess the only reason why I think you'd need to unset it is if something fails during the attempt. Almost like `serviceState` could be one of `unserved`, `serving`, `served`.